api/organisations/models.py

```diff
@@ -415,16 +415,26 @@ def _get_subscription_metadata_for_chargebee(self) -> ChargebeeObjMetadata:
         return cb_metadata
 
     def _get_subscription_metadata_for_self_hosted(self) -> BaseSubscriptionMetadata:
-        if not is_enterprise():
-            return FREE_PLAN_SUBSCRIPTION_METADATA
+        if is_enterprise() and hasattr(
+            self.organisation, "licence"
+        ):  # pragma: no cover
+            licence_information = self.organisation.licence.get_licence_information()
+            return BaseSubscriptionMetadata(
+                seats=licence_information.num_seats,
+                projects=licence_information.num_projects,
+                audit_log_visibility_days=None,
+                feature_history_visibility_days=None,
+            )
+        elif is_enterprise():  # pragma: no cover
+            return BaseSubscriptionMetadata(
+                seats=self.max_seats,
+                api_calls=self.max_api_calls,
+                projects=None,
+                audit_log_visibility_days=None,
+                feature_history_visibility_days=None,
+            )
```

Comment:
As discussed previously, I think we'll want to remove this eventually at least. Let's add a comment explaining the reasoning.

Reply:
Ok no problem I've added a comment.